docs/tutorials/migrate-from-neo4j.md

```diff
@@ -0,0 +1,497 @@
+MERGE (product:Product {productID: column.ProductID})
+  ON CREATE SET product.productName = column.ProductName, product.unitPrice = toFloat(column.UnitPrice);
+
+LOAD CSV WITH HEADERS FROM 'https://gist.githubusercontent.com/jexp/054bc6baf36604061bf407aa8cd08608/raw/8bdd36dfc88381995e6823ff3f419b5a0cb8ac4f/employees.csv' AS column
+MERGE (e:Employee {employeeID:column.EmployeeID})
+  ON CREATE SET e.firstName = column.FirstName, e.lastName = column.LastName, e.title = column.Title;
+
+CREATE INDEX product_id FOR (p:Product) ON (p.productID);
+CREATE INDEX product_name FOR (p:Product) ON (p.productName);
+CREATE INDEX employee_id FOR (e:Employee) ON (e.employeeID);
+CALL db.awaitIndexes();
+
+LOAD CSV WITH HEADERS FROM 'https://gist.githubusercontent.com/jexp/054bc6baf36604061bf407aa8cd08608/raw/8bdd36dfc88381995e6823ff3f419b5a0cb8ac4f/orders.csv' AS column
+MATCH (order:Order {orderID: column.OrderID})
+MATCH (product:Product {productID: column.ProductID})
+MERGE (order)-[op:CONTAINS]->(product)
+  ON CREATE SET op.unitPrice = toFloat(column.UnitPrice), op.quantity = toFloat(column.Quantity);
+
+LOAD CSV WITH HEADERS FROM 'https://gist.githubusercontent.com/jexp/054bc6baf36604061bf407aa8cd08608/raw/8bdd36dfc88381995e6823ff3f419b5a0cb8ac4f/orders.csv' AS column
+MATCH (order:Order {orderID: column.OrderID})
+MATCH (employee:Employee {employeeID: column.EmployeeID})
+MERGE (employee)-[:SOLD]->(order);
+
+LOAD CSV WITH HEADERS FROM 'https://gist.githubusercontent.com/jexp/054bc6baf36604061bf407aa8cd08608/raw/8bdd36dfc88381995e6823ff3f419b5a0cb8ac4f/employees.csv' AS column
+MATCH (employee:Employee {employeeID: column.EmployeeID})
+MATCH (manager:Employee {employeeID: column.ReportsTo})
+MERGE (employee)-[:REPORTS_TO]->(manager);
+```
+
+If you are going to use different dataset to migrate, be aware of the
+differences between Neo4j and [Memgraph data
+types](/reference-guide/data-types.md) (for example, Memgraph doesn't support
+`DateTime()` as there is no temporal type in Memgraph that supports timezones
+but you can modify data to use `localDateTime()`).
+
+## Exporting data from Neo4j
+
+To get your data out of Neo4j instance, use the Neo4j APOC export functionality.
+To install APOC, select the project, then in the right-side menu select *Plugins
+-> APOC* and press install.
+
+<img src={require('../data/tutorials/migrate-from-neo4j/install_APOC.png').default} className={"imgBorder"}/>
+
+Then enable export by setting the configuration flag `apoc.export.file.enabled`
+to `true` in the `apoc.config` file located in the `config` directory. To open
+the directory, select the active project, click on *...* -> *Open folder* ->
+*Configuration*. 
+
+Export the data into a CSV file using:
+
+```cypher
+CALL apoc.export.csv.all("shipping.csv", {})
+```
+
+Once exported, the file is located in Ne04j's *Import* folder. To open it,
```

Comment:
Fix typo **Ne04j's**.
Is this part necessary in the tutorial " To open it, select the active project, click on ... -> Open folder -> Import."? It seems that it is out of context.

Reply:
I don't understand why it would be out of context?
I'm instructing the user how to find the file they just exported. It's not very logical that it's in the Import folder + they have a shortcut as how to find it in your OS

Reply:
Why would one want to open it in Neo4j after the export? This part I don't understand.